buildkite/resource_team.go

```diff
@@ -120,6 +123,9 @@ func (t *teamResource) Schema(ctx context.Context, req resource.SchemaRequest, r
 				MarkdownDescription: "Whether members of the Team can create Pipelines.",
 			},
 		},
+		Blocks: map[string]resource_schema.Block{
+			"timeouts": timeouts.BlockAll(ctx),
+		},
```

Comment:
Ohh remember we were going to take the approach to use the global timeouts on the provider config instead of per resource. You can check out the cluster datasource for how to access the provider timeouts

Reply:
👍💪 - just altered